src/NetAppFiles/NetAppFiles/ChangeLog.md

```diff
@@ -18,7 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+Breaking change:
```

Comment:
Better remove this line as it does not start with a star (*). I am not sure if that will break the version bump-up script.